core/test/integration/CMakeLists.txt

```diff
@@ -222,6 +222,6 @@ add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.GasLimitE
 add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.GasLimitEstimationForDelegation COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.GasLimitEstimationForDelegation)
 add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.GasLimitEstimationForUnDelegation COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.GasLimitEstimationForUnDelegation)
 add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.GasLimitEstimationForRedelegation COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.GasLimitEstimationForRedelegation)
-add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.PendingUnbondings COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.PendingUnbondings)
-add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.PendingRedelegations COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.PendingRedelegations)
+#add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.PendingUnbondings COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.PendingUnbondings)
+#add_test (NAME ledger-core-integration-CosmosLikeWalletSynchronization.PendingRedelegations COMMAND ledger-core-integration-tests --gtest_filter=CosmosLikeWalletSynchronization.PendingRedelegations)
```

Comment:
This one can be fixed by running the operations on Obelix once every 3 weeks. The main question is : do we want to depend on external state for tests ? Do we want to mock http responses ? Do we have time to do that ?

Reply:
I prefer to not depend on manual actions for automatic tests. But if we can automate the obelix operations, why not.